lamindb/examples/mlflow/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""Examples and utilities for Mlflow.
+
+.. autosummary::
+   :toctree: .
+
+   save_mlflow_features
+"""
+
+import lamindb as ln
+
+
+def save_mlflow_features():
```

Comment:
I know this this one function and the wandb one are already in modules that carry the framework name but I still think that being explicit here makes it more readable.